### PR TITLE
add protocol, default http

### LINKED ISSUE #35: all personal websites are linked using http

### PR DESCRIPTION
**Pull request by Haroenv**
fixes #35

### CODE REVIEW

File: _data/members.yml [hunk 1]
```diff
@@ -54,7 +54,7 @@
 
 - name: Felix
   twitter: dervondenbergen
-  url: 'https://demont.is'
+  url: http://'https://demont.is'
```
Comment (dervondenbergen) [inline _data/members.yml]: what are you doing there :grin:

Comment (Haroenv) [inline _data/members.yml]: oh, didn't see you had that already :100:

**Comment by Haroenv** [comment]
oh and @dervondenbergen like it's now online, your link now shows `http://https//demont.is`, on my version it shows like the others

**Comment by dervondenbergen** [comment]
yes, but it looks good :shipit:

**Pull request merged**